src/routes/solid-router/reference/data-apis/cache.mdx

```diff
@@ -24,7 +20,35 @@ This cache accomplishes the following:
    Any user based navigation or link click bypasses it.
    Upon revalidation or new fetch the cache is updated.
 
-Using it with a load function:
+### Arguments
```

Comment:
Please move this section to after Examples, please. It should be a similar layout to [this one](https://docs.solidjs.com/reference/components/suspense-list)